### ISSUE TITLE
Unable to log-in to SaaS

### ISSUE THREAD

**Issue by vini-ppro** [aquasecurity/terraform-provider-aquasec#232]
Dear team, this issue is heavily impacting our migration timeline, your support is greatly appreciated.

Our instance has been migrated to the SaaS version, but the provider is now having issues to authenticate.

As explained [here](https://support.aquasec.com/support/solutions/articles/16000127855-how-to-authenticate-to-the-enterprise-api-in-the-cspm-platform), the flow to obtain the auth token is slightly different.

Following the documentation above, I can obtain a token from https://eu-1.api.cloudsploit.com/v2/signin via the API.

However the provider documentation [here](https://github.com/aquasecurity/terraform-provider-aquasec#using-the-aquasec-provider-saas-solution) says that `aqua_url` should still point to the `cloud.aquasec.com` domain. 

Getting a token from the `cloud.aquasec.com` is not possible via the API, which is probably what the provider is trying to do.

Can you please clarify how to authenticate with the SaaS version? This is the error I get:

```
│ Error: Unable to fetch token
│ 
│   with provider["registry.terraform.io/aquasecurity/aquasec"],
│   on provider.tf line 10, in provider "aquasec":
│   10: provider "aquasec" {
│ 
│ request failed. status: 401 Unauthorized, response: {"message":"You are not authorized to perform this action","code":401}
```

I have reviewed the credentials many times, they are correct.

Thanks.

**Comment by vini-ppro** [comment]
I have figured how to connect to the SaaS by looking at the code. 

I was using the wrong `cloud.aquasec.com` URL. I was using the URL to which I am redirected when I click the workload protection module, but I should be using the EU endpoint though: `https://eu-1.cloud.aquasec.com`

That became clear to me when I saw line: https://github.com/aquasecurity/terraform-provider-aquasec/blob/main/consts/consts.go#L5

I would suggest a small improvement in the documentation. Can we say something like: "The `aqua_url` should point to cloud.aquasec.com endpoint according to the list of official endpoints:". Then we provide the accepted endpoints from the list at https://github.com/aquasecurity/terraform-provider-aquasec/blob/main/consts/consts.go ?

Thanks.
